pkg/api/meta/meta.go

```diff
@@ -373,7 +414,8 @@ type genericAccessor struct {
 	labels            *map[string]string
 	annotations       *map[string]string
 	// TODO: This has to be a pointer to list, otherwise we can implement SetOwnerReferences
-	ownerReferences []unversioned.ObjectReference
+	// ownerReferences []*unversioned.ObjectReference
+	ownerReferences listAccessor
```

Comment:
maybe store this as a (ugh) reflect.Value and guarantee that it's the Slice type?